src/Domain/FlatLogicalMetric.hpp

```diff
@@ -0,0 +1,67 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace domain {
+
+/*!
+ * \brief The "flat logical metric" $\sum_k \frac{\partial x^k}{\partial
+ * \xi^\hat{i}} \frac{\partial x^k}{\partial \xi^\hat{j}}$, which is the flat
+ * spatial metric in element-logical coordinates.
+ *
+ * We define the "flat logical metric" to be the matrix of inner products of the
+ * $\hat{i}$-th logical coordinate basis vector $\partial_{\xi^\hat{i}}$ with
+ * the $\hat{j}$-th logical coordinate basis vector $\partial_{\xi^\hat{j}}$,
+ * where the inner product is taken assuming a flat spatial metric. When
+ * expressed in the ("inertial") $x$-coordinate system, each basis vector is a
+ * column of the Jacobian.
+ *
+ * When we need it, the computation domain metric in curved space can be
+ * computed just by a coordinate transformation of the spatial metric to logical
+ * coordinates.
```

Comment:
I don't understand what this paragraph is trying to tell me.

Reply:
Deleted since it's not important